app.rb

```diff
@@ -0,0 +1,73 @@
+require_relative 'person'
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'book'
+require_relative 'rental'
+
+class App
+  def list_books
+    Book.all.each { |book| puts "Title: \"#{book.title}\", Author: #{book.author}" }
+  end
+
+  def list_people
+    Person.all.each_with_index do |person, index|
+      puts "#{index}) Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"
+    end
+  end
+
+  def create_person
+    print 'Do you want to load a student (1) or a teacher (2) data? [Input the require number]: '
+    option = gets.chomp
+    print 'Name: '
+    name = gets.chomp
+    print 'Age: '
+    age = gets.chomp
+    if option == '1'
+      print 'Has parent permission [Y/N]: '
+      permission = gets.chomp
+      permission = permission.downcase == 'y'
+      print 'Classroom: '
+      classroom = gets.chomp
+      Student.new(age, classroom, name, parent_permission: permission)
+      puts 'Student created successfully'
+    else
+      print 'Specialization: '
+      specialization = gets.chomp
+      Teacher.new(age, name, specialization)
```

Comment:
- [ ] When creating a teacher, as soon as I enter the specialization of the teacher, the app crashes with the error message shown in the screenshot below. Could you work on it so a  teacher is created successfully? Please be sure all features work fine after fixing this issue before you submit it again for review 👍 

![image](https://user-images.githubusercontent.com/50941074/209413807-5025c8f5-cf13-40aa-b247-37628a460ee4.png)